rgwadmin/quota.go

```diff
@@ -0,0 +1,53 @@
+package rgwadmin
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// Quota struct
+type Quota struct {
```

Comment:
Here I guess quota scope is more user-specific than the bucket.  We can support for bucket later but IMO API name needs to change `Set/GetUserQuota`. Also, we need an API to enable quota. Otherwise, this won't work

Reply:
Ok I'll leave this for later then.